tianshou/policy/modelbased/psrl.py

```diff
@@ -163,15 +172,26 @@ class PSRLPolicy(BasePolicy):
 
     def __init__(
         self,
+        *,
         trans_count_prior: np.ndarray,
         rew_mean_prior: np.ndarray,
         rew_std_prior: np.ndarray,
+        action_space: gym.Space,
         discount_factor: float = 0.99,
         epsilon: float = 0.01,
         add_done_loop: bool = False,
-        **kwargs: Any,
+        observation_space: gym.Space | None = None,
+        action_scaling: bool = False,
+        action_bound_method: Literal["clip", "tanh"] | None = "clip",
+        lr_scheduler: TLearningRateScheduler | None = None,
     ) -> None:
-        super().__init__(**kwargs)
+        super().__init__(
+            action_space=action_space,
+            observation_space=observation_space,
+            action_scaling=action_scaling,
```

Comment:
can actually set action_scaling=False and action_bound_method=None here since psrl can be only applied on discrete case

Reply:
Done